cmd/ingester/app/consumer/consumer.go

```diff
@@ -149,11 +150,17 @@ func (c *Consumer) handleMessages(pc sc.PartitionConsumer) {
 				defer msgProcessor.Close()
 			}
 
-			msgProcessor.Process(&saramaMessageWrapper{msg})
```

Comment:
nit: it is much more efficient to pass `saramaMessageWrapper` by value

Reply:
Addressing this in https://github.com/jaegertracing/jaeger/pull/1818